index.js

```diff
@@ -20,6 +20,16 @@ var mainQueue = new TaskQueue();
 var tempViewQueue = new TaskQueue();
 var CHANGES_BATCH_SIZE = 50;
 
+var coffee;
+try {
```

Comment:
Background info on the why: https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#2-unsupported-syntax